Spaceport Culture (#5437)
endless-sky/endless-sky: data/human/culture free worlds.txt

			choice
				`	(Carry on with your business. I can always come back next year.)`
					defer
				`	(Stop and take in the sights - and maybe a soft pretzel or three.)`
			`	Conversation with the locals eventually turns to the matter of what they're celebrating in the first place. As it turns out, the bronzed pig holds the clue. Today is the anniversary of the Great Pig War, a conflict which apparently raged between the Southern Defense Pact and the Syndicate unbeknownst to the larger galaxy. Toasts are raised in honor of nine heroic captains - and to the memory of a single martyred swine.`

> 			`	"Back in the forties," a reveler drunkenly explains to you, "half the farms 'n' ranches hones' folks about this planet were either about ready to leave or takin' out a second mortgage on their property. Bankers from richer worlds were always comin' about to repossess something or another." A passer-by spits on the ground in disgust at the mention - or perhaps he'd just succeeded in extricating some shredded pork from his teeth. "So some good honest boys decided they'd had enough of payin' tariffs and honoring exclusive shipping rights n' rot like that. They came 'round, bought up a whole herd of swine, and took off to feed some hungry folks in the Syndicate. Trouble is, back in those days the Syndicate had a stranglehold on shipping grain n' livestock. Either you signed up with this one shipping company and let them eat all your profits, or you let your harvest rot. Getting to market without being a member was... somewhat frowned upon."`

Author: Honestly neither am I, this ones getting taken out and thrown back to Lake for the clean-up.

# Copyright (c) 2015 by Michael Zahniser
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE.  See the GNU General Public License for more details.

# "Missions" which add culture events to the spaceport.

mission "South: The Great Pig War"

> 	landing

Reviewer: Why `landing`? I'd have it minor and still in the spaceport, as it starts off talking about the spaceport.

Author: @Galaucus has some funny ideas. :P

Reviewer: that doesn't answer my question :james:

like I told WarlordMike recently, nobody here reviewing cares about spoilers, spell out what ideas are, get feedback now, see if that changes my mind or if I still think it makes more sense if it's a spaceport-triggered mission

Author: Since you missed it, - I tagged him because all your questions are on his content, I can't answer them.

Reviewer: I see, comment above lead me to believe you knew what the `funny ideas` were.

> 	landing

Reviewer 2: ```suggestion
	minor
```

> 	to offer
> 		and
> 			month == 6
> 			day >= 10
> 			day <= 20
> 	source
> 			attributes "spaceport" "farming" "south"

Reviewer 2: ```suggestion
	source
			attributes "farming" "south"
	to offer
		and
			month == 6
			day >= 10
			day <= 20
```

	on offer
		conversation
		`Farm world spaceports tend to be lonely affairs outside of harvesting season, but this one is bustling with festive energy. Rows of colorful tents and throngs of the planet's natives fill the area, as well as a proliferation of craftwork pig-sculptures; some are built from hay, some papier-mache and paint - one even appears to have been carved from butter. Most spaces that aren't taken by a silo, barn, or hangar of some sort have had a refreshment stand, game, or exhibition set up - the remaining few patches of ground are occupied by the feet of the planet's inhabitants. Live music fills the air, cheap beer flows like water, and fried food sizzles enticingly as it vacates vats of oil.`
		choice

> 			`	Carry on with your business. You can always come back next year.`

Reviewer: Action choices should be (Like this.)

> 				`	(Carry on with your business. You can always come back next year.)`

Reviewer 2: "Action" choices (those in parentheses) are always written in first person perspective.
Important to note though that we don't actually know that this is a yearly festival.
```suggestion
				`	(Carry on with your business. I can always come back next year.)`
```

> 					defer

Reviewer 2: I'd make this a decline thing, but that goes against the choice of "I can come back later," so maybe not.
```suggestion
					decline
```

Author: This is the start of a longer chain of conversations, so I think it would be a shame to make this a decline.

			`	Stop and take in the sights - and maybe a soft pretzel or three.`

> 		'Conversation with the locals eventually turns to the matter of what they're celebrating in the first place. As it turns out, the bronzed pig holds the clue. Today is the anniversary of the Great Pig War, a conflict which apparently raged between the Southern Defense Pact and the Syndicate unbeknowst to the larger galaxy. Toasts are raised in honor of nine heroic captains - and to the memory of a single martyrd swine.`

Reviewer: This starts with a ' instead of a `

Also, with the exception of the first line in a conversation, all other lines must be indented once.

> 			`	Conversation with the locals eventually turns to the matter of what they're celebrating in the first place. As it turns out, the bronzed pig holds the clue. Today is the anniversary of the Great Pig War, a conflict which apparently raged between the Southern Defense Pact and the Syndicate unbeknownst to the larger galaxy. Toasts are raised in honor of nine heroic captains - and to the memory of a single martyred swine.`

Reviewer 2: The southern defense pact is a more recent thing, and is also strictly about defending the south form pirates. Maybe change the two parties to just the southern worlds vs the Syndicate.

> 			`	"Back in the forties," a reveler drunkenly explains to you, "Half the farms'n'ranches'honesfolks about this planet were either about ready to leave or takin' out a second mortgage on their property. Bankers from richer worlds were always comin' about to repossess something or another." A passer-by spits on the ground in disgust at the mention - or perhaps he'd just succeeded in extricating some shredded pork from his teeth. "So some good honest boys decided they'd had enough of payin' tariffs and honoring exclusive shipping rights n' rot like that. They came 'round, bought up a whole herd of swine, and took off to feed some hungry folks in the Syndicate. Trouble is, back in those days the Syndicate had a stranglehold on shipping grain n' livestock. Either you signed up with this one shipping company and let them eat all your profits, or you let your harvest rot. Getting to market without being a member was... Somewhat frowned upon."`

Reviewer 2: ```suggestion
			`	"Back in the forties," a reveler drunkenly explains to you, "half the farms 'n' ranches hones' folks about this planet were either about ready to leave or takin' out a second mortgage on their property. Bankers from richer worlds were always comin' about to repossess something or another." A passer-by spits on the ground in disgust at the mention - or perhaps he'd just succeeded in extricating some shredded pork from his teeth. "So some good honest boys decided they'd had enough of payin' tariffs and honoring exclusive shipping rights n' rot like that. They came 'round, bought up a whole herd of swine, and took off to feed some hungry folks in the Syndicate. Trouble is, back in those days the Syndicate had a stranglehold on shipping grain n' livestock. Either you signed up with this one shipping company and let them eat all your profits, or you let your harvest rot. Getting to market without being a member was... somewhat frowned upon."`
```

			`	By this point several others have gathered around the speaker. It would appear that recounting the tale of the Great Pig War to strangers is part of the tradition. "... Mnh. S'anyways. Some boys flew over to Maker with a hundred-"`

> 			`	The speaker is interrupted by a shout of "It was a thousand!"`

Reviewer 2: ```suggestion
			`	The speaker is interrupted by a shout of, "It was a thousand!"`
```

> 			`	"Quiet down! Ain't no way you could fit a thousand swine onto a hauler. Alright, so they've got two hundred pigs in their hold when they try to land on Maker. Syndicate Security shows up-" Boos echo throughout the crowd "- and tell these good, upright, honest smugglers that they're to match velocity n' prepare to be boarded. Now, they aren't about to bow down, so they make a break for the edges of the system but start taking hits. Quicksilvers are buzzin', Protectors lumberin' after them, and their shields go down. A particle cannon punches straight through the hull, and our good, kind, noble, brave boys n' girls stop their ship and start patchin' the hole, ready to surrender."`

Reviewer 2: ```suggestion
			`	"Quiet down! Ain't no way you could fit a thousand swine onto a hauler. Alright, so they've got two hundred pigs in their hold when they try to land on Maker. Syndicated Security shows up-" Boos echo throughout the crowd "- and tell these good, upright, honest smugglers that they're to match velocity n' prepare to be boarded. Now, they aren't about to bow down, so they make a break for the edges of the system but start taking hits. Quicksilvers are buzzin', Protectors lumberin' after them, and their shields go down. A particle cannon punches straight through the hull, and our good, kind, noble, brave boys n' girls stop their ship and start patchin' the hole, ready to surrender."`
```

> 			`	"Quiet down! Ain't no way you could fit a thousand swine onto a hauler. Alright, so they've got two hundred pigs in their hold when they try to land on Maker. Syndicate Security shows up-" Boos echo throughout the crowd "- and tell these good, upright, honest smugglers that they're to match velocity n' prepare to be boarded. Now, they aren't about to bow down, so they make a break for the edges of the system but start taking hits. Quicksilvers are buzzin', Protectors lumberin' after them, and their shields go down. A particle cannon punches straight through the hull, and our good, kind, noble, brave boys n' girls stop their ship and start patchin' the whole, ready to surrender."`

Reviewer 3: hole*

> 			`	The listeners go quiet. Tense anticipation fills the air. If not for the jolly music and overwhelming clamor of the crowd surrounding your little huddle, all would be hushed. "And that's when nine green-striped angels punched outta' hyperspace and began streaking towards our little drama. See, some of the local militia decided that justice was more important than jurisdiction, and they decided t'give the smugglers an unasked-for escort. These fine folk arrived late to the party, after the shooting had already started, but just in time to make it stop. Syndicate security's surrounded, and..." People huddled around the speaker are grinning eagerly. They've heard this story a hundred times - and evidently this is their favorite part. "The bastards stand down! Our smugglers didn't make any profit on accounta' having to turn back, and one pig got rendered into bacon by laser fire, but folks got emboldened enough to start disregardin' contracts and began organizing their own shipping. Refused to use those crooks that were extortin' us as being the only way to get goods off planet. And that's why every year, on the anniversary of the Great Pig War, we celebrate."`

Reviewer 2: ```suggestion
			`	The listeners go quiet. Tense anticipation fills the air. If not for the jolly music and overwhelming clamor of the crowd surrounding your little huddle, all would be hushed. "And that's when nine green-striped angels punched outta' hyperspace and began streaking towards our little drama. See, some of the local militia decided that justice was more important than jurisdiction, and they decided t'give the smugglers an unasked-for escort. These fine folk arrived late to the party, after the shooting had already started, but just in time to make it stop. Syndicated Security's surrounded, and..." People huddled around the speaker are grinning eagerly. They've heard this story a hundred times - and evidently this is their favorite part. "The bastards stand down! Our smugglers didn't make any profit on accounta' having to turn back, and one pig out of the three hundred got rendered into bacon by laser fire, but folks got emboldened enough to start disregardin' contracts and began organizing their own shipping. Refused to use those crooks that were extortin' us as being the only way to get goods off planet. And that's why every year, on the anniversary of the Great Pig War, we celebrate."`
```

> 				decline

Reviewer 3: Maybe have the crowd clamor a little at the end of the narration.

mission "ZZCU South Culture: Repossessed"

> 	landing
> 	to offer
> 		and
> 			random < 2 + 50 * "Culture: Repossessed"
> 			has "event: war begins"
> 	source
> 		government "Free Worlds"
> 		attributes "spaceport" "farming" "south"

Reviewer 2: ```suggestion
	minor
	to offer
		and
			random < 2 + 50 * "Culture: Repossessed"
			has "event: war begins"
	source
		government "Free Worlds"
		attributes "farming" "south"
```

	on offer
		conversation
			`You arrive at the spaceport just in time to witness a shuttle being towed off its landing pad by a tractor, a familiar blazon displayed upon the spacecraft's doors. Though it's made somewhat harder to recognize by the fact that somebody has spray-painted "REPOSESED" over the logo, it appears that this shuttle belongs to - or used to belong to - the very same galactic bank that cut you the loan for your first ship. If the crowd of jeering locals is anything to go by, it looks like popular opinion here doesn't favor the galaxy's financial institution. A group of militia members observe the blatant ship-theft, evidently unwilling to intercede. In fact, some of them are cheering on the hijackers.`

> 				decline

Author: @Galaucus (this isn't mine)

> 		conversation
> 			`You arrive at the spaceport just in time to witness a shuttle being towed off its landing pad by a tractor, a familiar blazon displayed upon the spacecraft's doors. Though it's made somewhat harder to recognize by the fact that somebody has spray-painted "REPOSESED" over the logo, it appears that this shuttle belongs to - or used to belong to - the very same galactic bank that cut you the loan for your first ship. If the crowd of jeering locals is anything to go by, it looks like popular opinion here doesn't favor the galaxy's financial institution. A group of militia members observe the blatant ship-theft, evidently unwilling to intercede. In fact, some of them are cheering on the hijackers.`
> 				decline
> 	on decline
> 		"Culture: Repossessed" ++

Reviewer 2: ```suggestion
		"Culture: Repossessed" ++
		conversation
			`You arrive at the spaceport just in time to witness a shuttle being towed off its landing pad by a tractor, a familiar blazon displayed upon the spacecraft's doors. Though it's made somewhat harder to recognize by the fact that somebody has spray-painted "REPOSESED" over the logo, it appears that this shuttle belongs to - or used to belong to - the very same galactic bank that cut you the loan for your first ship. If the crowd of jeering locals is anything to go by, it looks like popular opinion here doesn't favor the galaxy's financial institution. A group of militia members observe the blatant ship-theft, evidently unwilling to intercede. In fact, some of them are cheering on the hijackers.`
				decline
```

		
mission "ZZCU Rim Culture: Repossessed"

> 	landing
> 	to offer
> 		and
> 			random < 2 + 50 * "Culture: Repossessed"
> 			has "event: war begins"
> 	source
> 		government "Free Worlds"
> 		attributes "spaceport" "urban" "rim"

Reviewer 2: ```suggestion
	minor
	to offer
		and
			random < 2 + 50 * "Culture: Repossessed"
			has "event: war begins"
	source
		government "Free Worlds"
		attributes "urban" "rim"
```

	on offer
		conversation

> 			`As you exit your ship, you run overhear a spaceport technician attempting to have an animated conversation with another. "Piracy? And the militia does nothing? That goes against everything we stand for." Stern disapproval colors the speaker's voice. "These clowns're assaulting bankers and taking their shuttles, last I heard. I get that they're frustrated out there on the farms and owe money, but that doesn't excuse theft." The technician's partner, currently shoulder-deep in an engine, responds distractedly. "Cool. You gonna' pass me that buffer or not? This Port and Polish ain't about to finish itself." The pair continue to converse as you carry on your way, but it's out of earshot before long.`

Reviewer 2: ```suggestion
			`As you exit your ship, you can overhear a spaceport technician attempting to have an animated conversation with another. "Piracy? And the militia does nothing? That goes against everything we stand for." Stern disapproval colors the speaker's voice. "These clowns 'er assaulting bankers and taking their shuttles, last I heard. I get that they're frustrated out there on the farms and owe money, but that doesn't excuse theft." The technician's partner, currently shoulder-deep in an engine, responds distractedly. "Cool. You gonna pass me that buffer or not? This port and polish ain't about to finish itself." The pair continue to converse as you carry on your way, but it's out of earshot before long.`
```

> 				decline

Author: @Galaucus (this isn't mine)

> 	on offer
> 		conversation
> 			`As you exit your ship, you run overhear a spaceport technician attempting to have an animated conversation with another. "Piracy? And the militia does nothing? That goes against everything we stand for." Stern disapproval colors the speaker's voice. "These clowns're assaulting bankers and taking their shuttles, last I heard. I get that they're frustrated out there on the farms and owe money, but that doesn't excuse theft." The technician's partner, currently shoulder-deep in an engine, responds distractedly. "Cool. You gonna' pass me that buffer or not? This Port and Polish ain't about to finish itself." The pair continue to converse as you carry on your way, but it's out of earshot before long.`
> 				decline
> 	on decline
> 		"Culture: Repossessed" ++

Reviewer 2: ```suggestion
	on offer
		"Culture: Repossessed" ++
		conversation
			`As you exit your ship, you run overhear a spaceport technician attempting to have an animated conversation with another. "Piracy? And the militia does nothing? That goes against everything we stand for." Stern disapproval colors the speaker's voice. "These clowns're assaulting bankers and taking their shuttles, last I heard. I get that they're frustrated out there on the farms and owe money, but that doesn't excuse theft." The technician's partner, currently shoulder-deep in an engine, responds distractedly. "Cool. You gonna' pass me that buffer or not? This Port and Polish ain't about to finish itself." The pair continue to converse as you carry on your way, but it's out of earshot before long.`
				decline
```
